src/utils.js

```diff
@@ -25,3 +25,11 @@ export let findKeys = _.curry((predicate, data) =>
 )
 
 export let coalesce = _.find(F.isNotNil)
+
+// ({ r: 'red', b: 'blue' g: 'green' }) -> Component<{ red, blue, green }> -> Component<{ red, blue, green, r, g, b }>
+export let withAliasProps = aliases => Component => props => (
```

Comment:
No need to reinvent the wheel:

https://github.com/acdlite/recompose/blob/master/docs/API.md#renameprops